README.md

```diff
@@ -200,17 +200,22 @@ apollo {
 You can get a schema.json file by running an introspection query on your endpoint. The Apollo Gradle plugin exposes a `downloadApolloSchema` task to help with this. You can download a schema by specifying your endpoint and the location where you want the schema to be downloaded:
 
 ```
-./gradlew :shared:downloadApolloSchema -Pcom.apollographql.apollo.endpoint=https://your.graphql.endpoint -Pcom.apollographql.apollo.schema=src/main/graphql/com/example/schema.json
+./gradlew :module:downloadApolloSchema -Pcom.apollographql.apollo.endpoint=https://your.graphql.endpoint -Pcom.apollographql.apollo.schema=src/main/graphql/com/example/schema.json
 ```
 
 If your endpoint requires authentication, you can pass query parameters and/or custom HTTP headers:
 
 ```
-./gradlew :shared:downloadApolloSchema -Pcom.apollographql.apollo.endpoint=https://your.graphql.endpoint -Pcom.apollographql.apollo.schema=src/main/graphql/com/example/schema.json  "-Pcom.apollographql.apollo.headers=Authorization=Bearer YOUR_TOKEN" "-Pcom.apollographql.apollo.query_params=key1=value1&key2=value2"
+./gradlew :module:downloadApolloSchema -Pcom.apollographql.apollo.endpoint=https://your.graphql.endpoint -Pcom.apollographql.apollo.schema=src/main/graphql/com/example/schema.json  "-Pcom.apollographql.apollo.headers=Authorization=Bearer YOUR_TOKEN" "-Pcom.apollographql.apollo.query_params=key1=value1&key2=value2"
 ```
 
 The `com.apollographql.apollo.headers` and `com.apollographql.apollo.query_params` properties both take a query string where key and values should be URL encoded.
 
+The default timeout for download operation is 1 minute. If you have a large `schema.json`, you may want to increase the timeout. Do that by adding the following into `gradle.properties`:
+```
+org.gradle.jvmargs=-DokHttp.connectTimeout=60 -DokHttp.readTimeout=60
```

Comment:
@silencecorner @tasomaniac Do you know if there's a specific reason to use system properties (-D) over project properties (-P) ? There's a bunch of both in the project and I feel it would be more clean to use the same everywhere but maybe not ?

Reply:
As far as I know, it is harder to put project properties as persistent. Like putting them in `gradle.properties`

I had this in mind while reviewing the prev PR. But I've seen other `-D` in the project. I didn't know we also have `-P`. I would also go for consistency.

Reply:
If wish there were a single way to do it :). 
I'll move the downloadSchema task config to `-D` so we have `-D` everywhere

Reply:
I saw you were using task inputs as parameters which works even without -P or -D.

Reply:
Ah yes, that's even better 👍

Reply:
I think we can have 2. Either that which looks much better. Or -D for the things that are more persistent. Like timeout setting.